source/testing/testing-models.md

```diff
@@ -77,24 +76,25 @@ export default Model.extend({
 });
 ```
 
-Then you could test that the relationship is wired up correctly
-with this test.
+Then you could test that the relationship by looking it up on the `user` model which it is part of.
 
 ```tests/unit/models/user-test.js
-import { moduleForModel, test } from 'ember-qunit';
-import { get } from '@ember/object';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { get } from "@ember/object"
 
-moduleForModel('user', 'Unit | Model | user', {
-  // Specify the other units that are required for this test.
-  needs: ['model:profile']
-});
+module('Unit | Model | user', function(hooks) {
+  setupTest(hooks);
+
+  test('should own a profile', function(assert) {
+    const User = this.owner.lookup('service:store').modelFor('user');
 
-test('should own a profile', function(assert) {
-  const User = this.store().modelFor('user');
-  const relationship = get(User, 'relationshipsByName').get('profile');
+    // lookup the relationship on the user model
+    const relationship = get(User, 'relationshipsByName').get('profile');
```

Comment:
why is this using `get(User, ...)` instead of `User.get(...)`?

Reply:
`DS.Model.get` is not a method (`User` here is a "class" not an instance).

Reply:
if there is no `get` then why not just `User.relationshipsByName.get('profile')`?

Reply:
Because I think `relationshipsByName` is a computed property on the constructor (yaaasss this is super weird [see here](https://github.com/emberjs/data/blob/7c6448d55efe441e9c62165ecc6bce9114ea4bcb/addon/-private/system/model/model.js#L1523))? I dunno, this kinda confuses me anyways, why do we actually need to enumerate all the relationships on this model?

Reply:
Like, IMHO, testing that this type of wiring works is super in the weeds. Ideally you make a profile instance, then make a user instance, then confirm that `user.get('profile') === profile;` or something.

Reply:
yeah, agreed